src/main/webapp/stylesheets/feedStyle.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:400,400i,700,700i,900,900i');
```

Comment:
you can probably name this file feed.css since the style is implied for all css files.